modules/go-health-monitoring.adoc

```diff
@@ -20,4 +20,6 @@ The {gitops-title} Operator will install the GitOps backend service in the `open
 . If the application is `out of sync` or `degraded` applications the respecting icons are displayed under the *Resources*. Hover over the icons to see the health status and the sync status. The icons are:
```

Comment:
```suggestion
. If the application is `out of sync` or `degraded`, icons are displayed under *Resources*. Hover over the icons to see health status and sync status. The icons are:
```
- Don't repeat `application`. It doesn't make sense.
- Word choice: `respecting` isn't the correct word. Do you mean `respective`? In any case, I don't think you need `respective` here, and you can safely omit it.
- omit `the` before `*Resources*`. By making it bold, you're already indicating to the user that it's a UI element.
- Definite articles aren't needed in the second sentence